documentation/src/main/asciidoc/topics/ref_discovery_aws.adoc

```diff
@@ -1,8 +1,9 @@
+[id='discovery_aws-{context}']
 = NATIVE_S3_PING
-On Amazon Web Service (AWS), use the `S3_PING` protocol for discovery.
+On Amazon Web Service (AWS), use the S3_PING protocol for discovery.
```

Comment:
```suggestion
On Amazon Web Service (AWS), use the NATIVE_S3_PING protocol for discovery.
```
warning, there is an S3_PING protocol as well. We don't ship it; we only ship the native one.